<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>intelligentInfer</class>
 <widget class="QDialog" name="intelligentInfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sample File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SampleFile_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="SampleFile_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>X Field</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="xFiled_comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Y Field</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="yFiled_comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Inference Property:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="InferedProperty_comboBox"/>
     </item>
     <item>
      <widget class="QCheckBox" name="RepresentativeSample_checkbx">
       <property name="text">
        <string>Representative sample set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Covariate Files:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddCov_btn">
       <property name="text">
        <string>Add Covariates</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="CovariateFiles_tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Output Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="OutputFile_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="OutputFile_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Inference_OK_btn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
